Ports/libgd/package.sh

```diff
@@ -3,9 +3,24 @@ port=libgd
 version=2.3.3
 useconfigure=true
 use_fresh_config_sub=true
-configopts=("--without-x")
+configopts=(
+    '--without-avif'
+    '--without-fontconfig'
+    '--without-freetype'
+    '--without-heif'
+    '--without-jpeg'
+    '--without-liq'
+    '--without-raqm'
+    '--without-tiff'
+    '--without-webp'
+    '--without-x'
+    '--without-xpm'
```

Comment:
My latest push includes all dependencies that can be used by `libgd`. I've added `--with-sysroot` to avoid the issue I was seeing 

Is it best to include all these dependencies? Given that the only thing that uses `libgd` at the moment is `npiet`, which only needs `libgd` for creating .png files. Including `fontconfig`/`freetype` seems a bit heavy-handed if we know it isn't going to be used by anything. The alternative would be to explicitly exclude all these optional dependencies that aren't currently being used then whoever needs them at a later date can enable them fairly easily.

Any thoughts on the best way to go here?

Reply:
As long as it's consistent regardless of the other ports that might have been installed by chance, either is probably fine. That's the only "real" issue with the situation, where most hard to debug issues would start.

How does `libgd` handle it's features when support has not been compiled/linked in? If it will result in compile-time linking errors if someone tries to use missing features, then disabling features is fine for now. If it's only detectable during run time, I'd say to just include those features to save others the headache.

Reply:
@tcl3 I was 95% certain that I already had libgd as a dependency for the PHP port, but that doesn't seem to be the case. I'd love to include it for `ext-gd` in the future, and in that case these extra dependencies will work very well!

Reply:
@gmta Fair enough, let's include all the dependencies we can.